tests/base/.eslintrc.json

```diff
@@ -0,0 +1,5 @@
+{
+  "root": true,
+  "extends": ["@bedrockstreaming", "@bedrockstreaming/eslint-config/typescript"],
+  "parserOptions": { "project": "./tests/base/tsconfig.json" }
```

Comment:
I don't know Typescript, but could parsers config be shared between our JS projects?

Reply:
I'm not sure sharing `tsconfig.json` file content would be relevant. But it doesn't concern `eslint-tools` anyway.